android/src/main/java/com/tuya/smart/rnsdk/activator/TuyaActivatorModule.kt

```diff
@@ -39,6 +38,13 @@ import com.facebook.react.bridge.WritableMap
 import com.facebook.react.modules.core.DeviceEventManagerModule.RCTDeviceEventEmitter
 import com.tuya.smart.sdk.api.*
 
+object ScanBeamConstants{
+  const val UUID = "uuid"
+  const val DEVICE_TYPE = "uuid"
+  const val MAC = "uuid"
+  const val ADDRESS = "uuid"
+}
+
```

Comment:
probably it should have different values.
also better to move it under `com.tuya.smart.rnsdk.utils.Constant` and import from there